cql3/expr/expression.cc

```diff
@@ -2277,15 +2287,19 @@ std::vector<managed_bytes_opt> get_elements(const cql3::raw_value& val, const ab
 }
 
 utils::chunked_vector<std::vector<managed_bytes_opt>> get_list_of_tuples_elements(const cql3::raw_value& val, const abstract_type& type) {
-    utils::chunked_vector<managed_bytes> elements = get_list_elements(val);
+    utils::chunked_vector<managed_bytes_opt> elements = get_list_elements(val);
     const list_type_impl& list_typ = dynamic_cast<const list_type_impl&>(type.without_reversed());
     const tuple_type_impl& tuple_typ = dynamic_cast<const tuple_type_impl&>(*list_typ.get_elements_type());
 
     utils::chunked_vector<std::vector<managed_bytes_opt>> tuples_list;
     tuples_list.reserve(elements.size());
 
-    for (managed_bytes& element : elements) {
-        std::vector<managed_bytes_opt> cur_tuple = tuple_typ.split_fragmented(managed_bytes_view(element));
+    for (managed_bytes_opt& element : elements) {
+        if (!element) {
+            // Note: just skipping is okay here, for our caller
```

Comment:
I don't understand this comment. It says skipping is okay, but doesn't skip...

Reply:
The intent was, if we skipped instead of throwing, it would be fine as well (IIRC). I can double check.

Reply:
Oh, I see, you meant to write something like "just skipping **would have also been** okay"